src/features/market-status/components/MarketStatus.tsx

```diff
@@ -24,7 +27,10 @@ export const MarketStatus: FC<MarketStatusProps> = ({ stock, keyStats }) => {
           Real-Time Price as of {dateFormat} EST
         </span>{" "}
         {keyStats && (
-          <span className={marketSign(keyStats.open)}> {marketStatus}</span>
+          <span className={marketSign(keyStats.isUSMarketOpen)}>
+            {" "}
```

Comment:
Q: Is it not better to use CSS to create a space between this span and the market status as opposed to `{" "}`?